MapboxNavigation/RouteMapViewController.swift

```diff
@@ -105,9 +106,12 @@ class RouteMapViewController: UIViewController, NavigationViewDelegate {
     var annotatesSpokenInstructions = false
 
     override func loadView() {
+        super.loadView()
```

Comment:
@frederoni heh, I totally didn't catch this until I hit it, but LPT: If you override a `loadView()` method of a `UIViewController()`, you must _never_ [call super](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621454-loadview) in your overloaded implementation.